src/vue/common/AssetLogo.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <img
+    v-if="logoKey"
+    class="asset-logo asset-logo__image"
+    :src="assetLogoUrl"
+  >
+  <p
+    v-else
+    class="asset-logo asset-logo__code-abbr"
+  >
+    {{ assetCode | abbreviate }}
+  </p>
+</template>
+
+<script>
+import config from '@/config'
+
+export default {
+  name: 'asset-logo',
+  props: {
+    logoKey: { type: String, default: '' },
+    assetCode: { type: String, required: true },
+  },
+  computed: {
+    assetLogoUrl () {
```

Comment:
you already have such property in `asset.record`

Reply:
Updated